src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -1278,16 +1282,19 @@ await PreviewBuildIntegratedProjectActionsAsync(buildIntegratedProject, actions,
                 {
                     // ignore these
                 }
-                catch (Exception)
+                catch (Exception ex)
                 {
-                    Debug.Fail("Error finding repository");
+                    logger.LogWarning(
+                        $"Error finding repository for '{pair.Key.PackageSource.Source}:" +
```

Comment:
Everything else in PackageManagement should have resource strings already. This needs one also.